src/main/resources/view/TaskCard.fxml

```diff
@@ -2,36 +2,46 @@
-<?import javafx.scene.layout.AnchorPane?>
+<?import javafx.scene.layout.FlowPane?>
 <?import javafx.scene.layout.HBox?>
+<?import javafx.scene.layout.StackPane?>
 <?import javafx.scene.layout.VBox?>
 <?import javafx.scene.text.Font?>
 
-<VBox fx:id="taskCard" alignment="CENTER_LEFT" maxWidth="-Infinity" minWidth="-Infinity" prefWidth="785.0" xmlns="http://javafx.com/javafx/8.0.92" xmlns:fx="http://javafx.com/fxml/1">
+<VBox fx:id="taskCard" alignment="CENTER_LEFT" maxHeight="-Infinity" minHeight="-Infinity" minWidth="-Infinity" prefWidth="600.0" stylesheets="@../style/taskcardstyles/TaskCardBaseStyle.css" xmlns="http://javafx.com/javafx/8.0.92" xmlns:fx="http://javafx.com/fxml/1">
    <children>
-      <HBox alignment="CENTER_LEFT">
+      <StackPane>
          <children>
-            <Label fx:id="titleLabel" text="1. Tasking Title">
-               <font>
-                  <Font name="Segoe UI Semibold" size="30.0" />
-               </font>
-            </Label>
-            <AnchorPane maxWidth="1.7976931348623157E308" HBox.hgrow="ALWAYS">
+            <ImageView fx:id="pinImage" fitHeight="32.0" fitWidth="32.0" pickOnBounds="true" preserveRatio="true" StackPane.alignment="CENTER_RIGHT" />
+            <HBox alignment="CENTER_LEFT">
                <children>
-                  <ImageView fx:id="pinImage" fitHeight="32.0" fitWidth="32.0" layoutX="514.0" layoutY="7.0" pickOnBounds="true" preserveRatio="true" AnchorPane.rightAnchor="1.0">
-                     <image>
-                        <Image url="@../images/gold_star.png" />
-                     </image>
-                  </ImageView>
+                  <Label fx:id="titleLabel" maxWidth="1.7976931348623157E308" text="1. Tasking Title">
+                     <font>
+                        <Font name="Segoe UI Semibold" size="30.0" />
+                     </font>
+                     <padding>
+                        <Insets bottom="4.0" />
+                     </padding>
+                  </Label>
+                  <Label fx:id="typeLabel1" text="Event" textFill="WHITE">
+                     <font>
+                        <Font name="Segoe UI Semilight" size="16.0" />
+                     </font>
+                     <padding>
+                        <Insets left="8.0" right="8.0" />
+                     </padding>
+                     <HBox.margin>
+                        <Insets left="8.0" />
+                     </HBox.margin>
+                  </Label>
                </children>
-            </AnchorPane>
+            </HBox>
          </children>
-      </HBox>
-      <HBox alignment="CENTER_LEFT">
+      </StackPane>
+      <HBox fx:id="tagsBox" alignment="CENTER_LEFT">
          <children>
-            <Label fx:id="typeLabel" alignment="TOP_LEFT" style="-fx-background-color: #00A4FF; -fx-background-radius: 100;" text="Event" textFill="WHITE">
+            <Label fx:id="typeLabel0" alignment="TOP_LEFT" text="Event" textFill="WHITE">
```

Comment:
They are "event/task" labels in two different locations, so I don't think so.
For tag labels, I think I have a way (issue raised).